### ISSUE TITLE
Keyboard focus is not moving to the 'search' edit field present in the report screen in Windows.

### ISSUE THREAD

**Issue by msftedad** [databrothers-code-storage/text-search-slicer#4]
**Test Environment:** 
OS: Windows 11 Enterprise Insider Preview Version 24H2(OS Build 26100.4061) 
Browser Edge Version: 139.0.3405.86 (Official build) (64-bit)
URL: https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-singhpragy_microsoft_com/ES_LQ1lncH1Jm4LZGw9Me44Bkbvf6bMqzrNS4Rh2e0SoIg?e=K3xfVP

**User Impact:**
Users relying on keyboard navigation cannot move focus to the 'search' edit field on the report screen in Windows, preventing them from entering or modifying search queries without using a mouse.

**Steps to Repro:**

1. Open the URL: https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-singhpragy_microsoft_com/ES_LQ1lncH1Jm4LZGw9Me44Bkbvf6bMqzrNS4Rh2e0SoIg?e=K3xfVP in edge browser.
2. 'Power BI Text Slicer' report will open. 
3. Navigate to the 'Search' section using tab key.
4. Verify whether keyboard focus is moving to the 'search' edit field present in the report screen or not.

**Actual Result:**
Keyboard focus is not moving to the 'search' edit field present in the report screen in Windows.

**Refer Attachment:**
Keyboard focus is not moving to the 'search' edit field present in the report screen in Windows.png
Keyboard focus is not moving to the 'search' edit field present in the report screen in Windows.mp4

https://github.com/user-attachments/assets/277ab8e0-0d2a-416d-a5a8-e0a91d6c6f23
<img width="967" height="571" alt="Image" src="https://github.com/user-attachments/assets/c816a8b4-0272-43bc-993e-b714d88dc3c7" />

**Expected Result:**
Keyboard focus should move to the 'search' edit field present in the report screen in Windows.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
